mlir/include/mlir/Dialect/Linalg/IR/LinalgRelayoutOps.td

```diff
@@ -266,7 +280,11 @@ def Linalg_UnPackOp : Linalg_RelayoutOp<"unpack"> {
     dimensions. If specified, it must have `n - k` elements. If specified, this
     permutation is applied before combining any dimensions.
 
-    Example:
+    Note, the amount of elements in the source (packed tensor) and the result
+    (unpacked) can be unequal, i.e. `SizeOf(source) >= SizeOf(result)`. As
+    the unpack operation may drop any padding introduced by the pack operation.
```

Comment:
1. I'd keep this shorter and favour code over text.
2. Personally, I feel that it makes more sense to say that the packing op may drop elements and, as an outcome of that, the resulting tensor might be smaller than the source tensor. 
3. I suggest more explicit `NumElementsOf` instead of `SizeOf` ("size" is a bit context-sensitive).
```suggestion
    Note,  the unpack operation may drop any padding introduced by the pack operation and hence the following holds `NumElementsOf(source) >= NumElementsOf(result)`. 
```